components/member-list-item/index.js

```diff
@@ -1,19 +1,17 @@
-import { string } from 'prop-types';
-import Link from 'next/link';
 import { getDataURL, getImgURL } from 'helper-functions/urls';
+import { useEffect, useRef } from 'react';
+
+import Link from 'next/link';
+import SocialMediaIcon from '../social-media-icon';
 import classNames from './member-list-item.module.scss';
-import { useRef, useEffect } from 'react';
+import { string } from 'prop-types';
 import useFetch from 'custom-hooks/useFetch';
 
 const PreviewMember = (props) => {
   const { rdsId } = props;
   const { data } = useFetch(getDataURL(rdsId));
   const imgRef = useRef();
 
-  const onClick = (e) => {
-    e.stopPropagation();
-  };
-
   useEffect(() => {
```

Comment:
are we missing fall back?

Reply:
nope

Reply:
what if no `data` and `rsId` is coming?

Reply:
i am not getting what u r saying here. Please explain urself and how do u want me to handle it.

Reply:
@Shihab-Duzno The contract from the API dictates that these fields will have to be present. If the API is not sending it, then we need to catch that on the backend side